ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/PoolFactory.java

```diff
@@ -32,6 +35,9 @@ public class PoolFactory {
   private static final UInt64 DEFAULT_HISTORICAL_SLOT_TOLERANCE = UInt64.valueOf(320);
   private static final int DEFAULT_MAX_ITEMS = 5000;
   private static final int DEFAULT_MAX_BLOCKS = 5000;
+
+  private static final int DEFAULT_MAX_SYNC_COMMITTEE_MESSAGES = 512;
```

Comment:
should we reference VALID_SYNC_COMMITTEE_MESSAGE_SET_SIZE constant instead of heaving another constant?